packages/cozy-harvest-lib/src/components/TriggerManager.jsx

```diff
@@ -276,11 +400,13 @@ export class TriggerManager extends Component {
           )}
           {step === 'accountForm' && (
             <>
-              <BackButton onClick={this.showCiphersList}>
-                {t('triggerManager.backToCiphersList')}
-              </BackButton>
+              {!account && (
```

Comment:
Maybe we can be more explicit here by introducing a `isUpdate = !account` variable?